src/commands/utility/assignMeTaskThread.js

```diff
@@ -0,0 +1,83 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { DISCORD_SERVER } = require('../../config');
+const { translateLanguage } = require('../../languages/index');
+
+const ASSIGN_EMOJI = '⚔';
+const ASSIGNED_PATTERN = /\|⚔ @[\w-]+\|/;
+const userAssignedPattern = (username) =>
+  new RegExp(`\\|${ASSIGN_EMOJI} \\@(${username})\\|`, 'g');
+
```

Comment:
I think this part is repeated on the code we can reuse it instead in line 13 and line 38
```suggestion
const generateAssignedThreadName = (username) => `|${ASSIGN_EMOJI} @${username}|`
```